reconcile/queries.py

```diff
@@ -583,7 +583,13 @@ def get_aws_accounts(
         ecrs=ecrs,
         cleanup=cleanup,
     )
-    return gqlapi.query(query)["accounts"]
+    accounts = gqlapi.query(query)["accounts"]
+    if terraform_state:
```

Comment:
What about integrations that use this query and don't care about tf state? I don't have any examples but was that considered?

Reply:
If the integration doesn't care about the `terraformState` then there isn't a problem :)